src/Components/FitnessFilters.js

```diff
@@ -145,12 +149,11 @@ class FitnessFilters extends React.Component {
           </View>
           <Text style={filterStyle.filterTitle}>Duration: {durationInMinutes || '-'} mins</Text>
           <Slider
-            minimumValue={1}
+            minimumValue={10}
```

Comment:
maybe use dynamic value for min and max ? like max and min value of available drills ?

Reply:
I think depends on the question the user is answering like you said in your message above